.github/workflows/pr.yaml

```diff
@@ -44,3 +44,33 @@ jobs:
 
       - name: ${{ matrix.engine }}-build
         run: make ${{ matrix.engine }}-build
+
+  release-build:
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Install operator-sdk
+        # Instructions from https://sdk.operatorframework.io/docs/installation/#install-from-github-release
+        run: |
+          export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+          export OS=$(uname | awk '{print tolower($0)}')
+          export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.33.0
+          curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+          chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
+
+      - name: release-build
+        env:
+          IMAGE_TAG_BASE: localhost:5000/janus-idp/operator
+        run: make release-build
```

Comment:
I suppose after this change we can reuse the make file here: https://github.com/janus-idp/operator/blob/main/.github/workflows/pr-docker-build.yaml#L98-L102 instead of my version of the script.

Note that you could use `OS=$(go env GOOS) && ARCH=$(go env GOARCH)` instead of parsing and tweaking `uname`, which might be cleaner at least for x64.

Reply:
> I suppose after this change we can reuse the Makefile's `release-build` target here: [`main`/.github/workflows/pr-docker-build.yaml#L98-L102](https://github.com/janus-idp/operator/blob/main/.github/workflows/pr-docker-build.yaml?rgh-link-date=2024-01-03T15%3A14%3A02Z#L98-L102) instead of my version of the script.

Yes, it would be great to reuse the Makefile target. That would allow us to check that the issue reported in #111 does not occur again in the future.

Reply:
My concern with this is that if I'm not using the `docker/build-push-action@v5` action, then I can't easily set `labels: ${{ steps.meta.outputs.labels }}` for `quay.expires-after=14d`. 

So maybe we need to update the Makefile to use `--label quay.expires-after=14d` for all the container image builds?